packages/charts/src/charts/table/header-row.component.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { HeaderCeil, ArrowsType, SortByType } from '@keen.io/ui-core';
+import { FormatFuncType } from '../../types';
+import { Header } from './table.styles';
+
+type Props = {
+  data?: any;
```

Comment:
Why the `data` is optional here ? What's more type should be more precise -> `Record<string, any>[]`;